src/main/java/se/sundsvall/supportmanagement/integration/db/model/CommunicationAttachmentEntity.java

```diff
@@ -0,0 +1,135 @@
+package se.sundsvall.supportmanagement.integration.db.model;
+
+import java.util.Objects;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+
+
+@Entity
+@Table(
+	name = "communication_attachment",
+	uniqueConstraints = {@UniqueConstraint(name = "UK_communication_attachment_data_id", columnNames = {"communication_attachment_data_id"})})
+public class CommunicationAttachmentEntity {
+
+	@Id
+	@Column(name = "attachment_id")
+	private String attachmentID;
+
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "communication_id", nullable = false, foreignKey = @ForeignKey(name = "fk_communication_attachment_communication_id"))
+	private CommunicationEntity communicationID;
```

Comment:
Might want to change the naming, "communicationID" might be missleading during development?